bank-sdk/sdk/src/main/res/values/strings.xml

```diff
@@ -16,7 +16,7 @@
     <string name="gbs_digital_invoice_items">Artikel</string>
     <string name="gbs_digital_invoice_what_is_this_button">Was ist das?</string>
```

Comment:
[Suggestion] Line 17 looks like artifacts that should be removed since we've deleted the info box.

Reply:
Great catch. I would do it in this ticket: https://ginis.atlassian.net/browse/PIA-3741